### ISSUE TITLE
Tabs, TaNav, TaTitle and Tab - Ensure content is visible to assistive technologies - (2036570287)

### ISSUE THREAD

**Issue by dqateam** [Esri/calcite-components#5606]
### Module: 
 32 Tabs, Tab Nav, Tab Title and Tab 

### Violation: 
 Ensure content is visible to assistive technologies 

### WCAG Reference: 
- [1.1.1 Non-text Content](https://www.w3.org/WAI/WCAG21/Understanding/non-text-content.html)
- [4.1.2 Name, Role, Value](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html) 

### Instance ID: 
 2036570287 

### Severity: 
 10 

### Description: 
 [Issue]
There are elements that are hidden for assistive technology users. Examples included:

- Rain content
- Snowy Rain Mix Content
- Thunderstorms Content

[User Impact]
Sighted keyboard-only users will not understand what this element is and will perceive it as an unnecessary tab stop.

[Code Reference] 
 ```
<calcite-tab id="calcite-tab-title-c6b12df2-2ac7-0c36-5cf3-cc2fcaed62a8" scale="m" calcite-hydrated="" aria-labelledby="calcite-tab-title-af4c882e-8899-59cb-dea1-cf6bc2da1fc5" active="" selected="">Rain Content</calcite-tab>
 ``` 

### Note: 
 Recommendation]
Developer should ensure that content is available for all assistive technology users.

[Complaint Code Example]
``` <calcite-tab id="calcite-tab-title-c6b12df2-2ac7-0c36-5cf3-cc2fcaed62a8" scale="m" calcite-hydrated="" aria-labelledby="calcite-tab-title-af4c882e-8899-59cb-dea1-cf6bc2da1fc5"> ```Rain Content``` </calcite-tab> ```
 

### Media Type: 
 Page Structure 

### Additional Resources: 
- Report Source: Product accessibility evaluation conducted on the Calcite Design System in August, 2022.
